INSTALL.txt

```diff
@@ -1,4 +1,50 @@
-To install this package, type the following in the command line:
 
-  python setup.py install
+Installation steps for Ubuntu/Debian
+=====================================
+
+(Note that we are using version 3.4 of python).
+
+
+sudo apt-get install -qq -y python3-pyqt5 python3-flake8
+
+git clone --depth=1 --branch=python3-ufo3 https://github.com/trufont/robofab
```

Comment:
unless for developers, I'm not sure it's a good idea for end-users to install the trufont's dependencies from their git repositories, let alone from master/HEAD.
What if one pushes something to, say, fonttools which inadvertently breaks something else in trufont?

I think it would be better to use a `requirements.txt` file, or maybe two of them: a "stable" one with packages frozen at specific commits which are known to be working; and a 'dev-requirements.txt` file which would clone from HEAD.

This way one could do `pip install -r requirements.txt` and pip would automatically grab all the dependencies.

A cool thing about pip + requirements.txt files, is that one can specify a git URL for a package (when that is not already available on PyPI) -- even a specific commit or tag:

E.g.:
```bash
$ echo 'git+http://www.github.com/behdad/fonttools.git@894a52a90fd2670a6d01af8ef0f9967bfc682be3#egg=fonttools' > requirements.txt
$ pip install -r requirements.txt
```

This would install the fonttools package from its github repo at a specific git commit hash.

More info here: https://pip.readthedocs.org/en/1.1/requirements.html